src/services/encryptionService.js

```diff
@@ -0,0 +1,38 @@
+import api from "@api/api";
+import initVodozemac, { Account } from "vodozemac-javascript";
+import { setEncryptedUser } from "@src/store/values/EncryptedUser";
+
+class EncryptionService {
+  constructor() {
+    this.vodozemac = initVodozemac();
```

Comment:
I assume we do not need to store promise into `this.vodozemac` since we aren't going to use this var ?

Probably simply do it this way would be better:
```
 initVodozemac().then(() => {
     console.log("Vodozemac init Ok");
 });
```